db/migrate/20220912190906_create_food.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFood < ActiveRecord::Migration[7.0]
+  def change
+    create_table :foods do |t|
+      t.string :name
+      t.string :measurement_unit
```

Comment:
As on the wireframe, please consider adding the `Quantity` column as well. And even  in the validation Food controller

Reply:
#### I have made those changes, pls review them again.